content/docs/reference/go-reference.md

```diff
@@ -3,7 +3,7 @@ title: "Go Buildpack Reference"
 weight: 300
 menu:
   main:
-    parent: Reference
+    parent: reference
```

Comment:
I think the other ones need to be changed to be capitalized no?

Reply:
other way around. we've got some new goodies in the `config.toml` that defines the top-level keys.